src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlDatabaseAuditing.md

```diff
@@ -0,0 +1,122 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+ms.assetid: 14814BF3-51AF-4E51-A8A6-661825BD88D1
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmSqlDatabaseAuditing
+
+## SYNOPSIS
+Gets the auditing settings of an Azure SQL database.
+
+## SYNTAX
+
+```
+Get-AzureRmSqlDatabaseAuditing [-ServerName] <String> [-DatabaseName] <String>
+ [-ResourceGroupName] <String> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzureRmSqlDatabaseAuditing** cmdlet gets the auditing settings of an Azure SQL database.
+To use the cmdlet, use the *ResourceGroupName*, *ServerName*, and *DatabaseName* parameters to identify the database.
+
+## EXAMPLES
+
+### Example 1: Get the auditing settings of an Azure SQL database
+```
+PS C:\>Get-AzureRmSqlDatabaseAuditing -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -DatabaseName "Database01"
+DatabaseName           : database01
+AuditAction            : {}
+AuditActionGroup       : {SUCCESSFUL_DATABASE_AUTHENTICATION_GROUP, FAILED_DATABASE_AUTHENTICATION_GROUP,
+                          BATCH_COMPLETED_GROUP, ...}
+ResourceGroupName      : resourcegroup01
+ServerName             : server01
+AuditState             : Enabled
+StorageAccountName     : mystorage
+StorageKeyType         : Primary
+RetentionInDays        : 0
+```
+
+## PARAMETERS
+
+### -DatabaseName
+SQL Database name.```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: True
+Position: 2
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+The name of the resource group.```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ServerName
+SQL Database server name.```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Confirm
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: cf
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -WhatIf
+Shows what would happen if the cmdlet runs. The cmdlet is not run.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: wi
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+## OUTPUTS
+
```

Comment:
Please make sure output types get in here - once this is hand edited, platyPS doesn't update it anymore